package.json

```
@@ -92,7 +92,8 @@
     "**/remark-parse/trim": "1.0.1",
     "**/typescript": "4.7.4",
     "**/underscore": "^1.13.1",
-    "globby/fast-glob": "^3.2.11"
+    "globby/fast-glob": "^3.2.11",
+    "wrap-ansi": "7.0.0"
```

The changes here may have been unintentionally backported.
```suggestion
    "globby/fast-glob": "^3.2.11"
```

---

For context, the resolution for `wrap-ansi` was added to circumvent failing build where the ESM version of the package was being required. [See here](https://buildkite.com/elastic/kibana-pull-request/builds/181741#018c39e5-617d-4e87-bed1-d259d67f1c8c)